server/src/main/scala/controllers/MockedChromaSendReceive.scala

```diff
@@ -26,6 +26,7 @@ trait MockedChromaSendReceive extends WithSendAndReceive {
                 |"token_type":"bearer","expires_in":86399}""".stripMargin))
 
         case Uri.Path(chromaPayh) if chromaPayh.contains("/chroma/live/") =>
+          log.info("Sending mocked chroma flights.")
```

Comment:
Want this log message?

Reply:
yeah, I think we can keep it - this won't be used except in testing.